docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to Contribute
+
+Please refer to the [readme](https://github.com/tablexi/nucore-open/blob/master/README.md) for spin-up information and basic context. 
```

Comment:
I'm pretty sure you can use relative paths, so `[readme](../README.md)`. Then if you're in any of the forks, you'll stay in that fork rather than getting sucked up into open.